docs/README-Extending.md

```diff
@@ -0,0 +1,79 @@
+# Extending Zowe Explorer
+
+The Zowe Explorer provides methods to enable third party extensions to interface with Explorer entities enriching the user experience.
+
+## Extender Package
+
+Extender packages are created as new VSCode extensions. Details for which can be found via the links below
+
+- [Using extensions in Visual Studio Code](https://code.visualstudio.com/docs/introvideos/extend) 
+- [VSCode Extension API](https://code.visualstudio.com/api) 
+- [vscode-extension-samples](https://github.com/Microsoft/vscode-extension-samples) 
+
+When creating an extension for Zowe Explorer it is necessary to specify that dependency in the package.json file as follows:
+```json	
+    "extensionDependencies": [
+		"zowe.vscode-extension-for-zowe"
+	],
+
+```
+The addition of this script also enforces that the Zowe Explorer is a pre-requisite package on the users machine which will install it if necessary 
+when the extender package is installed. This also ties the activate functionality of both packages which enables access to the Zowe Explorer API
+
+## Contextual Hooks
+
+To enable menus, extenders can reference the context values associated with the individual Tree Items. Zowe Extender uses a strategy of adding and removing 
```

Comment:
"Zowe Extender" -> "Zowe Explorer"

Reply:
I redeveloped the documentation as https://github.com/zowe/vscode-extension-for-zowe/pull/720. This was really here as a place-holder. Sorry for the confusion.